R/Lab 1.Rmd

```diff
@@ -0,0 +1,197 @@
+---
```

Comment:
If you are referring to the "Build & Reload" button, I have clicked it and
reloaded the project.  Unfortunately I still received the same error
indicating that it could not fetch imagepath and that "openBinaryFile: does
not exist (no such file or directory).  I will keep looking into this and
see if I can figure it out.

On Thu, May 5, 2016 at 11:16 AM, David Blodgett <notifications@github.com>
wrote:

> In R/Lab 1.Rmd
> <https://github.com/dblodgett-usgs/NWCEd/pull/5#discussion_r62220016>:
>
> > @@ -0,0 +1,197 @@
> > +---
>
> I think that once you've built the package it should work. Do you remember
> how to do that?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/dblodgett-usgs/NWCEd/pull/5/files/1b41e74bc8209dc38bef1127b4a9e7cf9235845c#r62220016>
>



-- 
Jake Nelson